src/components/events/event_info/fields/EventFieldView.css

```diff
@@ -28,4 +28,11 @@
 
 td:hover .initEditBtn {
     opacity: .4;
-}
\ No newline at end of file
+}
+
+.priority-btn {
+    border: none !important;
+    background: none !important;
+    outline:none !important;
+    box-shadow: none !important;
```

Comment:
Yikess!!!! A better approach is to do more specific selectors (say, `div.priority-btn` or `button.priority-btn`) and that sometimes fixes it. Or use an id: `#priority-btn`.

Reply:
Those are good suggestions :). I suppose I'll leave this to Morgan?

Reply:
Yupp... I will see what I get to anyways lolll